### PR TITLE
Add BKChem

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://gitlab.com/bkchem/bkchem/-/merge_requests/5
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://gitlab.com/bkchem/bkchem/-/commit/0265d4a2df941935a3c1972c02a3e21c8c6e9efb https://gitlab.com/bkchem/bkchem/-/merge_requests/5

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot build org.zirael.bkchem.BKChem

**Comment by Mailaender** [comment]
bot, build org.zirael.bkchem.BKChem

**Comment by Mailaender** [comment]
bot, build org.zirael.bkchem.BKChem

File: bkchem128.png
Comment (hfiguiere) [inline bkchem128.png:1]: you don't need this size since it's not upstream

Comment (Mailaender) [reply bkchem128.png:1]: It is upstream https://gitlab.com/bkchem/bkchem/-/commit/0265d4a2df941935a3c1972c02a3e21c8c6e9efb

**Comment by Mailaender** [comment]
bot, build org.zirael.bkchem.BKChem

File: bkchem128.png
Comment (hfiguiere) [reply bkchem128.png:1]: why not installing the SVG instead and ignore these sizes?

**Pull request closed without merging**